lalrpop-util/Cargo.toml

```diff
@@ -3,13 +3,13 @@ name = "lalrpop-util"
 description = "Runtime library for parsers generated by LALRPOP"
 repository = "https://github.com/lalrpop/lalrpop"
 license = "Apache-2.0 OR MIT"
-version = "0.19.9" # LALRPOP
+version = "0.19.11" # LALRPOP
 authors = ["Niko Matsakis <niko@alum.mit.edu>"]
 workspace = ".."
 edition = "2018"
 
 [dependencies]
-regex = { version = "1", optional = true }
+regex = { version = "1", features = ["std", "unicode"], optional = true }
```

Comment:
Should we apply the discipline proposed in https://github.com/lalrpop/lalrpop/issues/775#issuecomment-1523312554? That is, enable those features at the level of the workspace, and don't enable any feature at the crate-level to be sure that they won't get messed by a specific crate of the LALRPOP workspace changing configuration?

Reply:
Did you see #783? I was realizing that I'm not sure how these different levels of feature enabling intersect with lalrpop's features. Of course I'm all for turning everything on, but I just want to make sure that people who use lalrpop without the lexer still get whatever minimal setup they are looking for.

Reply:
I dropped this part and applied #783